openapi.yml

```diff
@@ -175,7 +142,7 @@ paths:
 
   /organizations/{ipaCode}/signed-documents:
     post:
-      description: Send via email the registration documents of the organization
+      description: Signs and sends via email the onboarding documents of the organization
```

Comment:
Maybe the endpoint could become something like `/onboarding-emails` instead? I'm considering this because with the current implementation this endpoint signs the documents for the requests whose ids are provided in the body, and send them as attachments with a single email. If we specify a single request id in the path and remove the request ids from the body, then it's not possible anymore to send a single email containing all the desired documents, instead two different emails will have to be sent. Is this the expected behavior?

Reply:
This a good point :) Another proposal:

`POST /requests/actions`

Payload contains request_id(s) + action type:

`{"type":"send-email", "ids":[1,2,3....]}`

ref. https://restful-api-design.readthedocs.io/en/latest/methods.html#actions
ref. https://stackoverflow.com/questions/21616669/rest-updating-multiple-records/37582468